dao/src/main/java/org/thingsboard/server/dao/attributes/CachedAttributesService.java

```diff
@@ -132,20 +133,21 @@ public ListenableFuture<Optional<AttributeKvEntry>> find(TenantId tenantId, Enti
     @Override
     public ListenableFuture<List<AttributeKvEntry>> find(TenantId tenantId, EntityId entityId, String scope, Collection<String> attributeKeys) {
         validate(entityId, scope);
-        attributeKeys.forEach(attributeKey -> Validator.validateString(attributeKey, "Incorrect attribute key " + attributeKey));
```

Comment:
You may replace all the changes with simple line like this, right?
```
attributeKeys =  new HashSet<>(attributeKeys);
```

Reply:
I do not want to override input variable, because it may confuse other developers in the future.

Reply:
Ok